script.reddit.reader/resources/ytdl_sites_porn

```diff
@@ -0,0 +1,101 @@
+3movs.com
```

Comment:
It is a list of domains that have video that youtube.dl can successfully parse. 
This way, the addon can let the user know that a link can be played.